xml/art_migration.xml

```diff
@@ -530,10 +530,14 @@
     <itemizedlist>
      <listitem>
       <para>
-       On each &slsa; compute node, remove all repositories that point to the
-       old deployer. To do this, run <command>zypper lr -u</command> and
-       remove any listings referring to the original deployer's IP address
+       On each &slsa; compute node, remove outdated repositories that point
+       to the old deployer. To do this, run <command>zypper lr -u</command> and
+       remove listings referring to the original deployer's IP address
        using <command>zypper rr <replaceable>REPO_NAME</replaceable></command>.
+       Note that the repositories <systemitem>SLES-OS<systemitem> and
```

Comment:
you need to close \<systemitem\> with \</systemitem\> to fix the validation error

Reply:
Done! Wouldn't you know it, the one time I don't run xmllint...